### ISSUE TITLE
First Flight

### ISSUE THREAD

**Issue by Sepper80** [IamPete1/Lynchpin#2]
Hi Pete,
first - thank you very much for your work and the great construction videos. I was there from the beginning and I am awaiting each of your videos, as I find flight characteristics of this thing fascinating.
I still had some FPV-Racecopter/Wing Hardware here and have finished the Lynchpin so far, I have watched each of your videos several times :-)

My Hardware:
Matek765 FC, 2x Matek GPS (one with Compass), 2x T-Motor Velox V2 4in1 ESC 45/55A (i use 3 channels per esc), 6x T-Motor Velox 1950kV, FrSky x-xsr & the Zoe 3D Props.

Since now everything works fine, 4 times checked parameters and settings... motors working, gps & wifi working, all calibrated, double tested...data field, horizon, messages  - all seems good.

But I have a few problems:

When I start, go to 5-10% throttle its out of control :-(, some motors run to full throttle, some stops and so on.
I think it´s a problem with the direction/propeller rotating. Do you have a better overview of how the propellers should be arranged? I see it in the videos...first I thought that all the propellers must "blow backwards" in the direction of the arrows on your markings, but in the videos it looks like the propeller on motor 4 is the other way?

I also can´t see FRAME_CLASS 16 ....don´t have the choice 16, (video 4, 14:25min) i can type in 16, then in the message board it shows 6dof scripting/ i think all ok? dev 4.1 flashed, the two scripts over mavftp are there. Do I have to start a script manually here?

BRD_SAFETYENABLE - i doesn´t have this option in my param list, i this important?

Fport - Sticks calibrated, all working. But on the yaapu telemetry script (i have the radiomaster t16s/horus screen) there´s no "action". i get all the sensors, ardu messages, yaapu seems ok, no erros - but horizon/compass/flightmodes still do nothing.
it almost looks like a first input is coming, which then freezes. on the MP Data screen all works. different Baud rates checked, invereted options checked...but always the same freezes :-(
Sholud i choose another firm on the x-xsr and go over s.port/sbus?

ESC Telemetry...works, but on battery monitor is always the half Volts (11,49) Battery is a normal 6s. Voltage calibration not accepted, no chance:-( bug in MP?. I go to Analog Voltage Sensor - all ok.

Sorry for all the Questions :-) My last Arducopter built was in 2014.
Thanks a lot 
Matthias

**Comment by IamPete1** [comment]
> When I start, go to 5-10% throttle its out of control :-(, some motors run to full throttle, some stops and so on.
> I think it´s a problem with the direction/propeller rotating. Or should i only give more Throttle?! :-)
> Do you have a better overview of how the propellers should be arranged? Motor directions ok, I see it in the videos...first I thought that all the propellers must "blow backwards" in the direction of the arrows on your markings when i test in MP on 55, but in the videos it looks like the propeller on motor 4 is the other way?

Yeah this defiantly sounds like you have a motor ordering or rotation issue. The props go on to blow past the motors, just as you would on a normal quad.

>I also can´t see FRAME_CLASS 16 ....don´t have the choice 16, (video 4, 14:25min) i can type in 16, then in the message board it > shows 6dof scripting/ i think all ok? dev 4.1 flashed, the two scripts over mavftp are there. Do I have to start a script manually > here?

ctrl + f and then 'param gen' will grab you the latest param descriptions in MP. If its showing 6dof scripting then your all good. Scripts start on boot, that you were even able to arm show the scripts are working.

> BRD_SAFETYENABLE 
doesn't matter, some boards are setup for a safety switch others are not.

Not really sure on the Fport and ESC telem, that is standard AP stuff so you could always have a look on the forum.

**Comment by Sepper80** [comment]
Hi Pete,

i checked the motors once again...motor4 false direction. 

My Lynchpin flys!!:-)

First Test, very stable.
Second Test - Upside down...lot of wobbling, burned one Motor on start :-(

I think the T Motor Velox are the prob, i had left the motors and not used them in my racers because they caused vibrations compared to good T Motor Fseries. Ordering a set of F40 and give it another try ;-)
About the fport - i think its a matek765 prob, when i read the forum and do some more tests(different settings and uarts). i think i use the sbus/s.port.

Thanks a lot Pete!

**Comment by JiangNanGenius** [comment]
Actually i'm using the same motor hope mine will not burn🤣🤣🤣🤣🤣

**Comment by Sepper80** [comment]
V1 Velox or V2? i used the older V1, golden version. 
really not a masterpiece from t-motor, really cheap and way apart from the quality and smooth running of other series.
yesterday i installed the f40, much smoother and i saved almost 40gr (one motor) in total. test flight the next few days.

@IamPete1 
Telemetry with Yaapuu works now too, flashed the x-xsr back an i use sbus/s.port.

**Comment by Sepper80** [comment]
![image](https://user-images.githubusercontent.com/40562635/109302252-4c175a80-7839-11eb-989b-4d49748d8f45.jpeg)

**Comment by JiangNanGenius** [comment]
Super Cool. I'm using V2, since it's super cheap🤣🤣🤣

**Comment by JiangNanGenius** [comment]
> V1 Velox or V2? i used the older V1, golden version.
> really not a masterpiece from t-motor, cheap but way apart from the quality and smooth running of other t-motor series.
> yesterday i installed the f40, much smoother and i saved almost 40gr (one motor) in total. test flight the next few days.
> 
> @IamPete1
> Telemetry with Yaapuu works now too, flashed the x-xsr back an i use sbus/s.port.

My motors are actually working well, but they seems to be not powerful enough. it seems the plane is a little bit heavy when i control it. Now i stuck on setting the slider to roll🤣🤣🤣🤣the most important function🤣🤣🤣🤣

**Comment by Sepper80** [comment]
Second test: stabi, althold, loiter works :) different start directions too ![image](https://user-images.githubusercontent.com/40562635/109534719-eab2ee00-7abb-11eb-8f4b-3fde4ea8c41b.jpeg)

**Comment by IamPete1** [comment]
Looking Good!

**Comment by JiangNanGenius** [comment]
Fantastic

**Comment by JiangNanGenius** [comment]
Now i get Lua: Panic: not enough memory when using 6DOF_All script. Mine have 512K ram. it seems  really strange.

**Comment by JiangNanGenius** [comment]
I solved that by changing the SCR_HEAP_SIZE to larger. My setting is 102400

**Comment by Sepper80** [comment]
the matek765 in my build has also 512k ram - no probs, changed nothing in the params. When did you got these error messages?

**Comment by JiangNanGenius** [comment]
> the matek765 in my build has also 512k ram / 2mb flash - no probs, changed nothing in the params. When did you got these error messages?

When you use 6DoF_all.lua you'll find this problem. The default setting of Ardupilot is only 43k. It's too small for the full script

**Comment by JiangNanGenius** [comment]
![20210302_160550.jpg](https://user-images.githubusercontent.com/79638157/109617327-38d5f900-7b71-11eb-9b2c-91cf3ef0e23e.jpg)
Success on the 3 modes too😃😃😃😃😃, but seems not stable enough🤣. Need more configuration. I tried to roll but the the direction gone when rolling back to 90 degrees. This field is too small🤣🤣🤣 and i just landed. tomorrow i'll find a larger place to try again.

**Comment by IamPete1** [comment]
@JiangNanGenius I notice you have the new version of the 3D props with the curved tips, I am using the older version with the square tips. I expect you can still get it to fly OK, but the tune will probably need some work.

**Comment by Sepper80** [comment]
I have both versions here, on the V2 prop i burned my velox v1 motor... im test it too. The v2 props has less pitch (3,5), the motor have to go higher and got hotter(indoor test) I Switch back to the v1 prop, but i test it out outdoors next time

**Comment by JiangNanGenius** [comment]
> I have both versions here, on the V2 prop i burned my velox v1 motor... im test it too. The v2 props has less pitch (3,5), the motor have to go higher and got hotter(indoor test) I Switch back to the v1 prop, but i test it out outdoors next time

![1614690017347.jpg](https://user-images.githubusercontent.com/79638157/109652161-608d8700-7b9a-11eb-95c3-5c3fb36e2470.jpg) So are thesw the props you use? 
![1614690236891.jpg](https://user-images.githubusercontent.com/79638157/109652488-d396fd80-7b9a-11eb-9c57-357df0588878.jpg)
i'm using these. And i ordered the other version. They should be arrive tomorrow

**Comment by Sepper80** [comment]
Yes, i use the older V1. But have the V2 in grey also here. ![image](https://user-images.githubusercontent.com/40562635/109656540-d37c0b00-7b64-11eb-9056-a59371441388.jpeg)

**Comment by IamPete1** [comment]
IMHO, pitch is just nonsense for a 3D prop, the blade has to be at 45 deg with no twist for it to be truly 3D, so all true 3D props should have the same pitch.

**Comment by Sepper80** [comment]
you are right pete, i think too much in normal racers :) ...the data sheets on gemfans website said 4 to 3.5 🤷🏻‍♂️

**Comment by JiangNanGenius** [comment]
i changed the props, it seems to be much better now. But another thing is when i started to roll in loliter mod it will move backwards, But in stablized and Althold that didn't happen.It seems strange.

**Comment by Sepper80** [comment]
my lynchpin holds the position in roll / pitch but overall it is too restless, far from the original.  almost as if my pids were way too high.  I had a little crash during the tests due to the extreme wobble, have to print a few new parts first :(

**Comment by IamPete1** [comment]
You could well have to tune I think your not running f40's allr round the same as I am? The Cube also has great damping internally its certainly worth checking the filtering with your flight controller.

https://ardupilot.org/copter/docs/common-imu-notch-filtering.html

**Comment by Sepper80** [comment]
I also switched to the f40.  the yaw is almost like a rc helicopter with  the gyro is set too high.  thanks for the link, i think it just needs some fine tuning.  the mpu6000 that is built into the matek actually works very, very well in quads, the board itself is also dampened again ... i have to see ... or switch to the cube.  is ins_hntch_mode 3 very important or can I also go to _mode 1?  my esc telemetry still only shows half the voltage, isn't that the wrong rpm value comes from the escs?

**Comment by IamPete1** [comment]
I'm sure you can get it flying just as well with the matek, you will just have to setup some of the stuff for it rather than going directly off my Cube param set. I found the BLHeli telem driven notch works great, but no doubt you can get close by manually setting up the throttle based notch. Its defiantly worth getting the filtering dialed in before messing with the tune too much.

**Comment by JiangNanGenius** [comment]
Mine is actually the same even with different motor
> I also switched to the f40.  the yaw is almost like a rc helicopter with  the gyro is set too high.  thanks for the link, i think it just needs some fine tuning.  the mpu6000 that is built into the matek actually works very, very well in quads, the board itself is also dampened again ... i have to see ... or switch to the cube.  is ins_hntch_mode 3 very important or can I also go to _mode 1?  my esc telemetry still only shows half the voltage, isn't that the wrong rpm value comes from the escs?

**Comment by Sepper80** [comment]
![image](https://user-images.githubusercontent.com/40562635/111347648-b163ac00-867f-11eb-915b-754f91de9d4a.jpeg) new parts ready, also in white:) waiting for better weather. Snow is back

**Comment by Sepper80** [comment]
A little feedback from me:
The new one flies in all modes, today I was briefly testing it in snow and -5 degrees outside. :-)

I looked again at the topic of damping, here I was able to reduce all values ​​(vibe log) by almost half by decoupling the carrier plate at the 3 points with alphagel. Before I had min / max 0-30 on all axes, now I really only wobble on yaw (Z axis), which you can also see in the log. I can't get it any better. The wobble on the yaw is of course extremely noticeable in flight, should I reduce yaw P here or increase the yaw rate with the damping? what do you mean pete? 
 - In any case, it looks better than before! Thanks a lot.

**Comment by Sepper80** [comment]
![image](https://user-images.githubusercontent.com/40562635/111874291-cf3d5380-8994-11eb-80c2-c071fd1497e2.png)

**Comment by Sepper80** [comment]
![image](https://user-images.githubusercontent.com/40562635/111874300-e7ad6e00-8994-11eb-8635-c7bfaeb3bea0.jpeg)

**Comment by Rickymaxwell** [comment]
Hello pete, curious on what version of Open TX companion you are using for the lynchpin...I'm using 2.3.11.
Thanks

**Comment by IamPete1** [comment]
@Rickymaxwell I'm on OpenTX 2.3.7, you found the models? They should just copy across I think.

**Comment by Rickymaxwell** [comment]
I'll update TX, thank you pete.

**Comment by Rickymaxwell** [comment]
Models?

**Comment by IamPete1** [comment]
@Rickymaxwell My OpenTX models are here, https://github.com/IamPete1/Lynchpin/blob/main/LynchPin.otx

**Comment by Rickymaxwell** [comment]
Hi pete, so upgrade  OTX 2.3.7 then download your model?
